### PR TITLE
Remove description prefixes from translated strings (1.14)

### PR DESCRIPTION
**Pull request by Wedge009**
Clean-up as described in #2965. See commit message for details.

### CODE REVIEW

File: po/wesnoth-tutorial/hr.po [hunk 1]
```diff
@@ -1754,7 +1754,7 @@ msgstr ""
 #, fuzzy
 #| msgid "female^Fighter"
 msgid "female^Warrior"
-msgstr "žena^Borac"
+msgstr "Borac"
```
Comment (stevecotton) [inline po/wesnoth-tutorial/hr.po]: Not an issue for this PR, but it seems Hungarian translates both the L1 and the L2 as the same word.

File: po/wesnoth/af.po [hunk 2]
```diff
@@ -1751,11 +1751,11 @@ msgstr "Moet $unit.name die drietandvurk op tel?"
 
 #: data/core/macros/items.cfg:514
 msgid "storm trident^Take it"
-msgstr "storm drietandvurk^los dit"
+msgstr "los dit"
```
Comment (stevecotton) [inline po/wesnoth/af.po]: Not an issue for this PR, but here "Take it" and "Leave it" are translated to the same phrase.

**Comment by soliton-** [comment]
Would be good to follow this up with an email to the i18n mailing list so the next translation updates don’t bring it all back.

File: po/wesnoth-anl/bg.po [hunk 3]
```diff
@@ -96,14 +96,14 @@ msgstr "Мал Шики"
...
-msgstr "teamname^Затворници"
+msgstr "Затворници"
 
 #. [side]
 #: data/multiplayer/scenarios/4p_A_New_Land.cfg:270
 #, fuzzy
 #| msgid "teamname^Prisoners"
 msgid "teamname^Monsters"
-msgstr "teamname^Затворници"
+msgstr "Затворници"
```
Comment (stevecotton) [inline po/wesnoth-anl/bg.po]: Same translation as "Prisoners", probably incorrect.

File: po/wesnoth-tutorial/hr.po (hunk 1 above)
Comment (Wedge009) [reply po/wesnoth-tutorial/hr.po]: The lines before this indicates is fuzzy - the previous msgid was Fighter. Leaving this as-is.

File: po/wesnoth/af.po (hunk 2 above)
Comment (Wedge009) [reply po/wesnoth/af.po]: Google says 'los dit' is 'leave it'. Clearing 'take it' as untranslated.

File: po/wesnoth-anl/bg.po (hunk 3 above)
Comment (Wedge009) [reply po/wesnoth-anl/bg.po]: Another fuzzy translation that needs to be updated - previously was 'Prisoners'. Leaving as-is.

**Comment by Wedge009** [comment]
@soliton- I forgot to ask earlier: who would be appropriate to make announcement? I could write something describing what we're doing here, but not sure if it's okay for me to send that.

**Comment by soliton-** [comment]
You could check with @ivanovic but I think it’s perfectly fine for you to send it.

**Comment by Wedge009** [comment]
Ivanovic said I should make the announcement. Will do so after merging.

**Pull request merged**